app/src/main/java/org/simple/clinic/settings/changelanguage/ChangeLanguageScreen.kt

```diff
@@ -138,9 +142,16 @@ class ChangeLanguageScreen(
   }
 }
 
-private class ChangeLanguageScreenUiActions(private val screenRouter: ScreenRouter) : UiActions {
+private class ChangeLanguageScreenUiActions(
+    private val screenRouter: ScreenRouter,
+    private val activity: Activity
+) : UiActions {
```

Comment:
Does this have to be a class? Can we just make the screen implement this interface instead?

Reply:
This can go in the next PR.

Reply:
I'll make this change.